### PR TITLE
feat(other): add locales check to backend and integrate it to test workflow

### LINKED ISSUE #2366: 💥 [DevOps][Backend] Have `yarn locales --fix` In The Backend
## 💥 DevOps ticket

Have `yarn locales --fix` In The Backend.
See `package.json` in frontend.

… if PR #2163 is merged.

### PR DESCRIPTION
**Pull request by mahula**
## 🍰 Pullrequest

The implementation of [email templates](https://github.com/gradido/gradido/pull/2163) introduced locales to the backend.
These require proper checks for local development and our CI worklfow - analogue to frontend and admin interface.

### Issues
- relates #2163 
- fixes #2366

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -360,6 +360,35 @@ jobs:
       - name: backend | Lint
         run: docker run --rm gradido/backend:test yarn run lint
 
+  ##############################################################################
+  # JOB: LOCALES BACKEND #######################################################
+  ##############################################################################
+  locales_frontend:
```
Comment (Elweyn) [inline .github/workflows/test.yml:366]: Should the name not be locales_backend instead of locales_frontend?
```suggestion
  locales_backend:
```

**Review by Elweyn — CHANGES_REQUESTED** [review]

File: .github/workflows/test.yml (hunk 1 above)
Comment (mahula) [reply .github/workflows/test.yml]: Thank you.
This is fixed in https://github.com/gradido/gradido/pull/2693/commits/ee5aea4c85dc85b6b42ac794c2edf1b52708379d.

**Review by Elweyn — DISMISSED** [review]

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -358,8 +358,37 @@ jobs:
       # LINT BACKEND ###########################################################
       ##########################################################################
       - name: backend | Lint
-        run: docker run --rm gradido/backend:test yarn run lint
+        run: docker run --rm gradido/backend:test pwd && yarn run lint
```
Comment (Elweyn) [inline .github/workflows/test.yml:361]:
```suggestion
        run: docker run --rm gradido/backend:test yarn run lint
```

File: .github/workflows/test.yml [hunk 3]
```diff
@@ -358,8 +358,37 @@ jobs:
...
+        with:
+          name: docker-backend-test
+          path: /tmp
+      - name: Load Docker Image
+        run: docker load < /tmp/backend.tar
+      ##########################################################################
+      # LOCALES FRONTEND #######################################################
+      ##########################################################################
+      - name: Backendend | Locales
+        run: docker run --rm gradido/backend:test yarn run locales
```
Comment (Elweyn) [inline .github/workflows/test.yml:390]:
```suggestion
        run: docker run --rm gradido/backend:test pwd && yarn run locales
```

**Review by ulfgebhardt — APPROVED** [review]

**Pull request merged**